static/app/views/settings/projectGeneralSettings/index.spec.tsx

```diff
@@ -352,7 +378,7 @@ describe('projectGeneralSettings', function () {
       expect(putMock).not.toHaveBeenCalled();
 
       // Click "Save"
-      await userEvent.click(screen.queryByRole('button', {name: 'Save'}));
+      await userEvent.click(screen.getByRole('button', {name: 'Save'}));
```

Comment:
good catch here! it would be awesome to get rid of the below `tick` as well 🙂

Reply:
Good eye @priscilawebdev, I removed it